### ISSUE TITLE
How to set min/max value for workshop settings?

### ISSUE THREAD

**Issue by nathan130200** [Zezombye/overpy#199]
When i use `createWorkshopSetting` this allow only set default value, but has no Min/Max value for Integer/Float types. When i scroll a little bit, this happen:
![image](https://user-images.githubusercontent.com/14365254/90634333-e600a980-e1fd-11ea-9074-fd1a086c01b4.png)

```cpp
#!define getSetting(Name, Type, DefaultValue) createWorkshopSetting(Type, "Combat Hud", Name, DefaultValue)

rule "Global settings":
    combatHudTimeout = getSetting("Target lock timeout", float, 10)
    combatHudDelay = getSetting("Target lock delay", float, 1.25)
```

Also don't have specific functions to create that.

This give to me a big float range:
`Workshop Setting Real(Custom String("Combat Hud", Null, Null, Null), Custom String("Target lock delay", Null, Null, Null), 1.25, -999999999999, 999999999999)`

Maybe is better separate functions, or create an optional param for int/float types when need min/max, and thrown error if bool is present and user try set min/max (because obviously have only true and false)

**Comment by Zezombye** [comment]
Right, this is in the wiki (https://github.com/Zezombye/overpy/wiki/Functions#workshop-settings) but I will put it in the argument description as it is more convenient.

As a general rule, you can decompile a function from overwatch, if you don't know how to represent a specific syntax :)

**Comment by nathan130200** [comment]
Oh thank you, i've not see in Wiki. I only see in intellisense hover text. Also i forgot the decompile feature sorry 😂

**Comment by Zezombye** [comment]
Keeping open as I want to add this to the intellisense.
